docs/pages/application-access/getting-started.mdx

```diff
@@ -178,11 +178,22 @@ Make sure to update `apps[0].name` and `apps[0].uri` accordingly if you're using
 
 ### Create a user
 
-Next, let's create a user to access the application we've just connected. Teleport has a built-in role called `access` that allows users to access cluster resources. Create a local user assigned this role:
+<ScopedBlock scope={["oss"]}>
+Next, let's create a user to access the application we've just connected. Teleport has a built-in role called `access` that allows users to access cluster resourcesCreate a local user assigned to these roles:
 
 ```code
 $ tctl users add --roles=access alice
 ```
+</ScopedBlock>
+
+<ScopedBlock scope={["enterprise", "cloud"]}>
```

Comment:
If it's important that Enterprise users not miss this part of the page, we should put this in a `Tabs` rather than a `ScopedBlock`, since users tend not to realize you can manipulate the scope switcher at the top of the page.

Reply:
I don't think it's super dire if they miss this.